pkg/filesystem/subvolume.go

```diff
@@ -175,8 +177,18 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 			}
 			// append the subvolume which doesn't have any snapshot attached to it.
 			for _, sv := range subvol {
-				state := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
+				state, err := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
+				// subvolume info returns error in case of pending clone or if it is not ready
+				// it is suggested to delete the pvc before deleting the subvolume.
+				if err != nil {
+					if errors.Is(err, syscall.EAGAIN) {
+						logging.Info("Subvolume info not found for: %q %q", sv.Name, err)
+						logging.Info("Please delete the pending pv if any before deleting the subvolume %s", sv.Name)
```

Comment:
they also need to scale down the cephfs csi deployment to avoid stale resources isn't it?

Reply:
added the log @Madhu-1